### PR TITLE
Add training queue view

### LINKED ISSUE #32: Add training queue view
Add view wherein staff can see a full list of members of a training queue.

### PR DESCRIPTION
**Pull request by keelandaye**
This PR adds a view wherein staff can inspect the full list of members in a given training queue, and allows them to remove members from the queue within that view.

Closes #32 
<hr />
e0e24abba155d1a71d516c1f4c512cce<br />
Updated action menu on "Manage Training Queues" page (with "View queue" option)
<hr />
fb222608ba374f0616c90b1b58008a8b
New training queue view page

### CODE REVIEW

File: src/routes/[id]/training/queues/[queueId]/+page.server.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import prisma from '$lib/prisma';
+import type { PageServerLoad } from './$types';
+
+export const load: PageServerLoad = async ({ params, depends }) => {
+    depends("queue:data");
+
+    console.log("Running load function...");
```
Comment (hhhapz) [inline src/routes/[id]/training/queues/[queueId]/+page.server.ts]: Probably can remove this print statement

File: src/routes/[id]/training/queues/[queueId]/data-table.svelte [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  import { readable } from "svelte/store";
+  import DataTableActions from "./data-table-actions.svelte";
+  import * as Table from "$lib/components/ui/table";
+  import { addPagination } from "svelte-headless-table/plugins";
+  import { Button } from "$lib/components/ui/button";
+  import type { TrainingQueueMembership } from "@prisma/client";
+
+  export let members: TrainingQueueMembership[];
+
+  const timeString = (joinedAt: Date): string => {
```
Comment (hhhapz) [inline src/routes/[id]/training/queues/[queueId]/data-table.svelte:17]: We have a similar function in `src/routes/admin/connections/+page.svelte`. I think we should just merge them into a single implementation and move it to $lib.

**Review by hhhapz — CHANGES_REQUESTED** [review]
looks good, just a few nit picky things on my end

File: src/routes/[id]/training/queues/[queueId]/+page.server.ts (hunk 1 above)
Comment (keelandaye) [reply src/routes/[id]/training/queues/[queueId]/+page.server.ts]: Good catch, have removed!

**Review by hhhapz — APPROVED** [review]

File: src/routes/[id]/training/queues/[queueId]/data-table.svelte (hunk 2 above)
Comment (keelandaye) [reply src/routes/[id]/training/queues/[queueId]/data-table.svelte]: Have done this! 278ea687720e8c0371c2e326582ecad9177137f9

**Pull request merged**